Android: Duplicate class org.walleth.khex.HexFunKt found in modules KHex-0.6.jar (com.github.komputing:KHex:0.6) and khex-0.6.jar (com.github.walleth:khex:0.6)

Thank you for creating this wonderful library.

I installed & linked `react-native-uport-signer: 1.3.3` and built a debug, the following error occurred.

```
> Task :app:checkDebugDuplicateClasses FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:checkDebugDuplicateClasses'.
> 1 exception was raised by workers:
  java.lang.RuntimeException: Duplicate class org.walleth.khex.HexFunKt found in modules KHex-0.6.jar (com.github.komputing:KHex:0.6) and khex-0.6.jar (com.github.walleth:khex:0.6)

  Go to the documentation to learn how to <a href="d.android.com/r/tools/classpath-sync-errors">Fix dependency resolution errors</a>.
```

I checked the gradle dependencies and it seems that both `react-native-uport-signer` and `uport-android-signer` have `kotlin-common:signer-common`, so when I removed the former, the build passed.

```
dependencies {
    implementation 'com.facebook.react:react-native:+'

    api "com.github.uport-project:uport-android-signer:0.3.1"
    api "com.github.uport-project.kotlin-common:signer-common:0.1.1"  // removed this line
}
```

Is the above line necessary?

Thank you.

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -43,6 +43,7 @@ dependencies {
     implementation 'com.facebook.react:react-native:+'
 
     api "com.github.uport-project:uport-android-signer:0.3.1"
-    api "com.github.uport-project.kotlin-common:signer-common:0.1.1"
+
+    testImplementation "junit:junit:4.12"
 }
   

